src/main/webapp/resources/js/pages/admin/components/user/CreateNewUser.jsx

```diff
@@ -0,0 +1,339 @@
+                  value={locale.language}
+                >
+                  {locale.name}
+                </Select.Option>
+              ))}
+            </Select>
+          </Form.Item>
+          <Form.Item
+            label={i18n("CreateNewUser.form.role.label")}
+            name="role"
+            rules={[
+              {
+                required: true,
+                message: i18n("CreateNewUser.form.role.required"),
+              },
+            ]}
+          >
+            <Select>
+              {systemRoles.map((role, index) => (
+                <Select.Option
+                  key={`create-new-user-account-role-${index}`}
+                  value={role.code}
+                >
+                  {role.name}
+                </Select.Option>
+              ))}
+            </Select>
+          </Form.Item>
+          {!emailConfigured && (
+            <Alert
+              style={{ marginBottom: SPACE_SM }}
+              message={i18n("CreateNewUser.emailConfigured.alert.title")}
+              description={
+                <Typography.Paragraph>
+                  {i18n("CreateNewUser.emailConfigured.alert.description")}
+                </Typography.Paragraph>
+              }
+              type="warning"
+              showIcon
+            />
+          )}
+          {mailFailure && (
+            <Alert
+              style={{ marginBottom: SPACE_SM }}
+              message={i18n("CreateNewUser.mailFailure.alert.title")}
+              description={
+                <Typography.Paragraph>
+                  {i18n("CreateNewUser.mailFailure.alert.description")}
+                </Typography.Paragraph>
+              }
+              type="error"
+              showIcon
+            />
+          )}
+          <Form.Item name="activate" valuePropName="checked">
+            <Checkbox
+              disabled={!emailConfigured || mailFailure}
+              onChange={(e) => setActivationEmail(e.target.checked)}
+            >
+              {i18n("CreateNewUser.form.activate.label")}
+            </Checkbox>
+          </Form.Item>
+          {!activationEmail && (
+            <>
+              <Alert
+                style={{ marginBottom: SPACE_SM }}
+                message={i18n("CreateNewUser.changePassword.alert.title")}
+                description={
+                  <Typography.Paragraph>
+                    <List
+                      header={i18n(
+                        "CreateNewUser.changePassword.alert.description"
+                      )}
+                      dataSource={passwordRules}
+                      renderItem={(item) => <List.Item>{item}</List.Item>}
+                    />
+                  </Typography.Paragraph>
+                }
+                type="info"
+                showIcon
+              />
+              <Form.Item
+                label={i18n("CreateNewUser.form.label.password")}
+                name="password"
+                rules={[
```

Comment:
This can be updated to use the `validatePassword` function in `validation-utilities.js`. You can go ahead and update the function to check for each of the patterns below and return the appropriate validation result. To see how the `validatePassword` function is used you can take a look at line 171-180 in `PasswordReset.jsx`

Reply:
Do we need an ajax endpoint to retrieve the password patterns? Or is this setting not changeable by instance admins.

Reply:
Change password under the under the security tab in user details was deployed first... but if we do share validate password could we split out a separate failure for each rule, rather than all rules being covered in one regular expression? That should make it easier for the user to figure out what they did wrong.

Reply:
Yup that's what I meant. You can update the validatePassword function to have the individual regexs and the appropriate validation messages returned

Reply:
Please see 51a57e911c191436c7c04826c9dfd1081b377509.

Reply:
One more thing kind of related to this. I see that we use the below for the password policy in multiple places. We should probably create a password policy alert component that we can just import into the components it is required

`const passwordRules = [
    i18n("validation-utilities.password.minimumLength"),
    i18n("validation-utilities.password.uppercase"),
    i18n("validation-utilities.password.lowercase"),
    i18n("validation-utilities.password.number"),
    i18n("validation-utilities.password.specialCharacters"),
  ];`

Reply:
Sure. I just need to ask when/if we're displaying the two password recommendation. In my user details PR I was asked to only display the password requirements and remove the password recommendations. Now I see the password recommendations got approved in the password reset page PR. Should the password recommendations always be displayed in this alert?

Reply:
Yes 100% make this a reusable component,

Reply:
Only have the required recommnedations

Reply:
So yeah go ahead and remove those 2 recommendations!

Reply:
Please see 06c958ca5dccb8469322916e14cecc6b256c7307.